packages/react/src/components/SignIn/SignIn.tsx

```diff
@@ -113,18 +115,22 @@ const SignIn: FC<SignInProps> = (props: SignInProps) => {
       throw new AsgardeoUIException('REACT_UI-SIGN_IN-HA-IV02', 'Auth response is undefined.');
     }
 
-    setIsComponentLoading(true);
+    authContext.setIsAuthLoading(true);
```

Comment:
Better to expose it directly from the context rather than using `authContext.setIsAuthLoading`